src/pages/users/Checkout.js

```diff
@@ -0,0 +1,176 @@
+import { useState, useEffect } from "react";
+import { Input } from "../../components";
+import axiosService from "../../services/configAxios";
+import { useNavigate } from "react-router";
+
+const CheckoutForm = () => {
+  const [carts, setCarts] = useState([]);
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [address, setAddress] = useState("");
+  const [payment, setPayment] = useState("");
+
+  const navigate = useNavigate();
+  const fetchData = async () => {
+    try {
+      const { data } = await axiosService.get("/shopping-cart");
+      setCarts(data.carts);
+      console.log(data.carts);
+    } catch (error) {
+      throw new Error("Error fetching data:", error);
+    }
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const handleSelect = (e) => {
+    console.log(e.target.value);
+    setPayment(e.target.value);
+  }
+  // debugger
+  const handelSubmitForm = async(e) => {
+    e.preventDefault();
+    console.log("Username:", name);
+    console.log("Email:", email);
+    console.log("Phone:", phone);
+    console.log("Address:", address);
+    console.log("Payment:", payment);
+    const redirect = 'redirect';
+    try {
+      const postData = await axiosService.post('/checkout', {name, email, phone, address, payment,user_id: 2 , totalPrice: 20000,redirect});
+      window.location.href=postData.data.data;
+      
+    } catch (error) {
+      throw new Error('Wrong went you checkout')
+    }
+  }
+ 
+
+  return (
+    <form className="form-checkout" onSubmit={handelSubmitForm}>
+      <div className="container">
+        <h3 className="checkout-heading">CHECKOUT PAGE</h3>
+        <hr style={{ borderWidth: "2px", borderColor: "#C4C4C4" }} />
+        <h4 className="checkout-heading text-center mt-4">Bill information</h4>
+
+        <div className="row">
+          <div className="form-container">
+            
+            <div className="form-group mb-3 col-md-6">
```

Comment:
Create new component for this one => 
 <div className="form-group mb-3 col-md-6">
              <label htmlFor="" className="form-label">Email</label>
              <input type="email" className="form-control" value={email}  onChange={(e) =>setEmail(e.target.value)} />
            </div>

Reply:
I have just resolved